package.json

```diff
@@ -23,17 +23,17 @@
     "@glazed/datamodel": "^0.2.0",
     "@glazed/devtools": "^0.1.3",
     "@glazed/did-datastore": "^0.2.0",
-    "@walletconnect/web3-provider": "^1.3.3",
-    "authereum": "^0.1.0",
-    "dag-jose": "^0.3.0",
-    "dids": "^2.1.0",
+    "@walletconnect/web3-provider": "^1.6.6",
+    "authereum": "^0.1.14",
+    "dag-jose": "^1.0.0",
+    "dids": "^2.4.0",
     "esbuild-loader": "^2.16.0",
     "fortmatic": "^2.2.1",
-    "key-did-resolver": "^1.2.0",
-    "multiformats": "~4.6.1",
-    "skynet-js": "^4.0.5-beta",
-    "uint8arrays": "^2.1.5",
-    "web3modal": "^1.9.3"
+    "key-did-resolver": "^1.4.0",
+    "multiformats": "^9.4.10",
+    "skynet-js": "^4.0.18-beta",
```

Comment:
I think this can be removed if we don't have the SkyDB example anymore?

Reply:
I didn't see anything that was using the SkyDB when I started, I think I want to hold this here for just a little longer until I get confirmation that SkyDB isn't being used (it's also why I'm holding off on Merging this PR right now.)